### PR TITLE
Feature/496 2

### PR DESCRIPTION
**Pull request by ddelpiano**
- HHModel mitigations for the kernel crashing, spinner and error dialogs.
- unit tests to reproduce the problem fixed.
- versions updated

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 setuptools.setup(
     name="netpyne_ui",
-    version="0.7.0",
+    version="0.8.0",
```
Comment (filippomc) [inline setup.py:18]: the version change should be a release branch thing. If this feature branch is derived from the release branch, it should be merged back to the release branch

**Review by filippomc — CHANGES_REQUESTED** [review]
Looks like tests are not passing, will wait for the green light.
It also seems that the branch is being derived from the release branch from the version numbering changes, so may make sense to change the base of this PR to the release branch.

File: setup.py (hunk 1 above)
Comment (enicolasgomez) [reply setup.py]: Rolled back as requested

**Review by enicolasgomez — APPROVED** [review]

**Review by enicolasgomez — APPROVED** [review]

**Comment by ddelpiano** [comment]
> Looks like tests are not passing, will wait for the green light. It also seems that the branch is being derived from the release branch from the version numbering changes, so may make sense to change the base of this PR to the release branch.

the test that is not passing is the new test based on padraig json, it is intended to not pass until we fix whatever needs to be changed on the backend (not now to be fair since we don't have enough time).
This will make it into development that will make it as well into the release branch since development contains other PRs that have been merged recently, is that fine?

**Review by filippomc — COMMENTED** [review]
As Nico approved, it's fine for me

**Review by filippomc — APPROVED** [review]

**Pull request merged**